admin_guide/diagnostics_tool.adoc

```diff
@@ -68,14 +71,77 @@ Non-standard locations can be specified with flags (respectively,
 `--config`, `--master-config`, and `--node-config`). If a configuration file
 is not found or specified, related diagnostics are skipped.
 
-Consult the output with the `--help` flag for all available options.
+Available diagnostics include:
+
+[options="header"]
+|===
+
+|Diagnostic Name |Purpose
+
+|AggregatedLogging
+|Check the aggregated logging integration for proper configuration and operation.
+
+|AnalyzeLogs
+|Check systemd service logs for problems. Does not require a configuration file to check against.
+
+|ClusterRegistry
+|Check that the cluster has a working Docker registry for builds and image streams.
+
+|ClusterRoleBindings
+|Check that the default cluster role bindings are present and contain the expected subjects according to base policy.
+
+|ClusterRoles
+|Check that cluster roles are present and contain the expected permissions according to base policy.
+
+|ClusterRouter
+|Check for a working default router in the cluster.
+
+|ConfigContexts
+|Check that each context in the client configuration is complete and has connectivity to its API server.
+
+|DiagnosticPod
+|Creates a pod that runs diagnostics from an application standpoint, which checks that DNS within the pod is working as expected and the credentials for the default service account authenticate correctly to the master API.
+
+|EtcdWriteVolume
+|Check the volume of writes against etcd for a time period and classify them by operation and key. This diagnostic only runs if specifically requested, because it does not run as quickly as other diagnostics and can increase load on etcd.
+
+|MasterConfigCheck
+|Check this host's master configuration file for problems.
+
+|MasterNode
+|Check that the master running on this host is also running a node to verify that it is a member of the cluster SDN.
+
+|MetricsApiProxy
```

Comment:
Api or API in MetricsApiProxy? Even though we normally write API, the Api in this value is likely correct, but just checking.

Reply:
Hmm this is what it says in the --help output, but I doublechecked, and the diagnostics runs, but needs a metrics to be configured. I'm gonna say it's ok if it gets that far.

Reply:
Right, it's a name and (perhaps unfortunately) it is case sensitive. Camel-casing acronyms in the middle of a name is common...